src/pages/[slug]/[ie]/index.tsx

```diff
@@ -366,13 +393,45 @@ const ObjectDetailPage: NextPage = () => {
 					src={playableUrl}
 					poster={mediaInfo?.thumbnailUrl || undefined}
 					title={representation.name}
-					pause={pauseMedia}
+					pause={isMediaPaused}
 					onPlay={handleOnPlay}
+					onPause={handleOnPause}
 					token={publicRuntimeConfig.FLOW_PLAYER_TOKEN}
 					dataPlayerId={publicRuntimeConfig.FLOW_PLAYER_ID}
+					plugins={['speed', 'subtitles', 'cuepoints', 'hls', 'ga', 'audio']}
 				/>
 			);
 		}
+		if (FLOWPLAYER_AUDIO_FORMATS.includes(representation.dctermsFormat)) {
+			if (!peakFileId || !!peakJson) {
+				return (
+					<div className={styles['c-audio-player-wrapper']}>
+						<FlowPlayer
+							className={clsx(
+								'p-object-detail__flowplayer',
+								showFragmentSlider && 'p-object-detail__flowplayer--with-slider'
+							)}
+							key={flowPlayerKey}
+							src={[
+								{
+									src: 'https://bertyhell.s3.eu-central-1.amazonaws.com/projects/test-files/road-to-joy.mp3',
```

Comment:
🔪

Reply:
klopt, is enkel om de PR review makkelijker te maken. Ik voeg een TODO toe om deze te wissen voor ik deze merge